mitoinstaller/mitoinstaller/installer_steps/mitosheet_installer_steps.py

```diff
@@ -61,7 +60,6 @@ def install_step_mitosheet_install_mitosheet():
         if error_traceback_last_line == 'Consider using the `--user` option or check the permissions.':
             do_user_install = input("The installer hit a permission error while trying to install Mito. Would you like to do a user install? Note that this will not work inside a virtual enviornment. [y/n] ")
```

Comment:
I wonder what this input does if the user gets to this step. I should trigger this on purpose!